e2e/content-services/search_multiselect.e2e.ts

```diff
@@ -0,0 +1,253 @@
+/*!
+ * @license
+ * Copyright 2016 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import TestConfig = require('../test.config');
+import AlfrescoApi = require('alfresco-js-api-node');
+
+import Util = require('../util/util');
+import resources = require('../util/resources');
+import CONSTANTS = require('../util/constants');
+import { UploadActions } from '../actions/ACS/upload.actions';
+
+import LoginPage = require('../pages/adf/loginPage');
+import SearchDialog = require('../pages/adf/dialog/searchDialog');
+import SearchResultPage = require('../pages/adf/searchResultsPage');
+import SearchFiltersPage = require('../pages/adf/searchFiltersPage');
+
+import AcsUserModel = require('../models/ACS/acsUserModel');
+import FileModel = require('../models/ACS/fileModel');
+
+describe('Search Component - Multi-Select Facet', () => {
+    let loginPage = new LoginPage();
+    let searchDialog = new SearchDialog();
+    let searchResultsPage = new SearchResultPage();
+    let uploadActions = new UploadActions();
+    let searchFiltersPage = new SearchFiltersPage();
+    let site;
+
+    beforeAll(() => {
+        this.alfrescoJsApi = new AlfrescoApi({
+            provider: 'ECM',
+            hostEcm: TestConfig.adf.url
+        });
+    });
+
+    describe('', () => {
+        let jpgFile, jpgFileSite, txtFile, txtFileSite;
+        let acsUser = new AcsUserModel();
+
+        let jpgFileInfo = new FileModel({
+            'location': resources.Files.ADF_DOCUMENTS.JPG.file_location,
+            'name': resources.Files.ADF_DOCUMENTS.JPG.file_name
+        });
+        let txtFileInfo = new FileModel({
+            'location': resources.Files.ADF_DOCUMENTS.TXT_0B.file_location,
+            'name': resources.Files.ADF_DOCUMENTS.TXT_0B.file_name
+        });
+
+        beforeAll(async (done) => {
+            await this.alfrescoJsApi.login(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
+
+            await this.alfrescoJsApi.core.peopleApi.addPerson(acsUser);
+
+            await this.alfrescoJsApi.login(acsUser.id, acsUser.password);
+
+            site = await this.alfrescoJsApi.core.sitesApi.createSite({
+                title: Util.generateRandomString(8),
+                visibility: 'PUBLIC'
+            });
+
+            jpgFile = await uploadActions.uploadFile(this.alfrescoJsApi, jpgFileInfo.location, jpgFileInfo.name, '-my-');
+            await browser.driver.sleep(5000);
```

Comment:
Please don't use browser.driver.sleep(). In this case, i think the file is uploaded in time anyway.